packages/fx-core/src/plugins/resource/aad/constants.ts

```diff
@@ -20,6 +20,16 @@ export class Constants {
   static statusCodeUserError = 400;
   static statusCodeServerError = 500;
 
+  static createOwnerDuplicatedMessage =
+    "One or more added object references already exist for the following modified properties: 'owners'.";
```

Comment:
This error message is very technical, we'd better use another words to general users such "xxx has already be added to the owner list"

Reply:
This error message is from Graph, and is used to match the scenario that we add a user who is already owner. Should be added when adding grant api. I will remove this line.

Reply:
Sure